src/wstool/multiproject_cmd.py

```diff
@@ -345,7 +345,9 @@ def do_work(self):
                                  verbose=self.report.verbose)
             return {}
 
-    work = DistributedWork(len(preparation_reports), num_threads, silent=False)
+    work = DistributedWork(len(preparation_reports),
+                           num_threads=num_threads,
+                           silent=False)
```

Comment:
To be consistent with the other calls to `DistributedWork` it would be `capacity=len(...`.